src/main/java/com/icthh/xm/ms/entity/service/impl/XmEntityServiceImpl.java

```diff
@@ -386,10 +387,21 @@ public XmEntity findOne(IdOrKey idOrKey) {
     @Override
     @Transactional(readOnly = true)
     public XmEntity findOne(IdOrKey idOrKey, List<String> embed) {
+
+        if (CollectionUtils.isEmpty(embed)) {
+            return findOne(idOrKey);
```

Comment:
in case when embed empty you should not fetch relations, but you fetch ALL relations!